### PR TITLE
Add documentation about mapped CLI e2e to do

### PR DESCRIPTION
**Pull request by danielbdias**
This PR adds a README on `testing/cli-e2etest` giving a brief explanation on what tests we can do with that engine.

## Changes

- Added a README on `testing/cli-e2etest`

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: testing/cli-e2etest/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Tracetest CLI e2e tests
```
Comment (jfermi) [inline testing/cli-e2etest/README.md:1]:
```suggestion
# Tracetest CLI e2e Tests
```

File: testing/cli-e2etest/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+In this folder we have the End-to-end tests done on the CLI to guarantee that the CLI is working fine. 
+The main idea is to test every CLI command against Tracetest server with different data stores and different Operating systems.
```
Comment (jfermi) [inline testing/cli-e2etest/README.md:4]:
```suggestion
The main idea is to test every CLI command against the Tracetest server with different data stores and different operating systems.
```

File: testing/cli-e2etest/README.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+
+## Implementation status
```
Comment (jfermi) [inline testing/cli-e2etest/README.md:7]:
```suggestion
## Implementation Status
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by mathnogueira — APPROVED** [review]

**Pull request merged**